src/reporter/tests/test_1T1E1A.py

```diff
@@ -200,6 +200,38 @@ def test_1T1E1A_lastN(reporter_dataset):
     assert_1T1E1A_response(obtained, expected)
 
 
+def test_1T1E1A_lastN_with_limit(reporter_dataset):
```

Comment:
is suppose with offset is not possible, correct? dumb question do we give feedback for "uncorrect" queries ?

Reply:
is this affecting only 1T1E1A?

Reply:
> is suppose with offset is not possible, correct? dumb question do we give feedback for "uncorrect" queries?

Which incorrect queries are you thinking specifically? It'd be weird to specify both offset and lastN (like lastN is so big you need to paginate the response), but I believe still valid, no?

> is this affecting only 1T1E1A?

Good question. Also for 1T1ENA, for which I can add a test now. For the more general endpoints (1TNE1A, 1TNENA, etc.) I believe lastN was not fully defined / implemented yet. We could discuss this tomorrow.

Reply:
As incorrect I meant, combinations of parameters that should not be allowed.

Reply:
In general, we have some checks and return some 400's when the request is really bad (see https://github.com/smartsdk/ngsi-timeseries-api/pull/250/commits/71dd4b1a6649e102ef79173c59426c240c637004 making this a bit better hopefully). For sure we don't cover all cases. It's easier to see when specific values are wrong, but if you have examples of combinations in mind, it'll be easier to check those specifically.

Reply:
not sure :) because wrong combination can lead to screw up sql that's all. not something also to fix in this pr, it was just a check